src/store/base.py

```diff
@@ -0,0 +1,56 @@
+from abc import ABC, abstractmethod
+import os
+from SPARQLWrapper import SPARQLWrapper, JSON, QueryResult
+
+class BaseStore(ABC):
+
+    # Every client runs the setup method
+    def __init__(self):
+        self.setup()
+
+    # Every client must have a setup method
+    @abstractmethod
+    def setup(self):
+        ...
+
+    @abstractmethod
+    def get_datasets(self):
+        ...
+
+
+class SparqlStore(BaseStore):
```

Comment:
can you put this in a different python file at the same level please.

i.e
- ./store/base.py - has BaseStore
- ./store/sparql.py - has SparqlStore

will make our lives easier when we start adding more store client later.